phpBB/phpbb/notification/method/base.php

```diff
@@ -104,6 +134,55 @@ public function add_to_queue(\phpbb\notification\type\type_interface $notificati
 	}
 
 	/**
+	* {@inheritdoc}
+	*/
+	public function update_notification($notification, array $data, array $options)
```

Comment:
email, jabber... any method which doesn't store the notification (and so can't interact with it)